sphinx_gallery/gen_rst.py

```diff
@@ -392,9 +409,9 @@ def scale_image(in_fname, out_fname, max_width, max_height):
                           generated images')
 
 
-def save_thumbnail(image_path, base_image_name, gallery_conf):
+def save_thumbnail(image_path, base_image_name, gallery_conf, thumb_num=1):
     """Save the thumbnail image"""
-    first_image_file = image_path.format(1)
+    first_image_file = image_path.format(thumb_num)
```

Comment:
You should rename the variable so that the thumbnail figure is not necessarily the first one.